src/components/DetailsOverlay/styles.ts

```diff
@@ -0,0 +1,15 @@
+import styled from "@emotion/styled";
+import { Subtitle } from "@leafygreen-ui/typography";
+import { size } from "constants/tokens";
+
+export const FilterRow = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: ${size.m};
+`;
+
+// @ts-ignore-error
+export const StyledSubtitle = styled(Subtitle as div)`
```

Comment:
idk how true this alias is but we should totally steal these from mms https://github.com/10gen/mms/blob/master/client/js/common/types/emotion.ts